src/main/scala/dotty/dokka/DottyDokkaConfig.scala

```diff
@@ -8,7 +8,8 @@ import collection.JavaConverters._
 import dotty.tastydoc.representations._
 
 case class DottyDokkaConfig(docConfiguration: DocConfiguration) extends DokkaConfiguration:
-  override def getOutputDir: String = new File("output").getAbsolutePath
+  var _outputDir: String = new File("output").getAbsolutePath
+  override def getOutputDir: String = _outputDir
```

Comment:
Sorry for this hack, but it is case class, and I needed to change outputDir in tests, otherwise it threw exception due to file overriding.